homework.md

```diff
@@ -44,67 +44,71 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+JavaScript is a Scripting language that was initially created to make your webpages alive.
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+It helps us to share information.
+
+It allows people to make thier own changes to other peoples repo, and share it back with them.
+
+It allows you to keep track with every change you made using checkpoints to your repo and if any mistake was done, you will know where you went wrong by checking your checkpoints.
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You are copying the owners repo to your repo list in your Github profile.
 
 ## Question 10 
 
 What happens when you clone a repostory?
 
-It send it to a friend
+It's when you are downloading an entire repo to your local machine.
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+You are telling the owner of the repo that you made some changes to thier repo.
```

Comment:
Hi Ulrich!

Awesome work! Just wanted to notify you of your minor typo here (line 75).

Reply:
Hi Sinopungula, lol i didn't even see that typo mistake, thank you very much.